Makefile

```diff
@@ -12,9 +12,11 @@ PKGS = $(shell go list ./... | grep -v /vendor/ | grep -v /tests/e2e)
 ARCH ?= $(shell go env GOARCH)
 BUILD_DATE = $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 GIT_COMMIT ?= $(shell git rev-parse --short HEAD)
+OS ?= $(shell uname -s | tr A-Z a-z)
 ALL_ARCH = amd64 arm arm64 ppc64le s390x
 PKG = github.com/prometheus/common
 GO_VERSION = 1.15.7
+PROM_VERSION = 2.24.1
```

Comment:
Do you want to be explicit and call the var PROMETHEUS_VERSION ?

Reply:
Sure